src/NHSD.BuyingCatalogue.Organisations.Api/Models/ApplicationUser.cs

```diff
@@ -45,13 +44,12 @@ private ApplicationUser()
             if (email is null)
                 throw new ArgumentNullException(nameof(email));
 
-            Id = userId.ToString();
-            UserName = userName?.Trim();
-            NormalizedUserName = UserName?.ToUpperInvariant();
-            FirstName = firstName?.Trim();
-            LastName = lastName?.Trim();
+            UserName = userName.Trim();
+            NormalizedUserName = UserName.ToUpperInvariant();
```

Comment:
Would it make sense to use the parameter consistently here? (Although there will obviously be no functional difference.)

Reply:
I didn't want to have to trim it again.